src/Orleans.Runtime/GrainTypeManager/GrainTypeData.cs

```diff
@@ -1,150 +0,0 @@
-using System;
```

Comment:
Class is deleted but has GlobalPlacement info, problematic?

Reply:
we need to add similar info which can be used in CachedGrainLocator.